src/mmw/sass/pages/_monitor.scss

```diff
@@ -0,0 +1,34 @@
+.monitor-window {
+  padding: 15px;
```

Comment:
Consider setting this to `1rem` so it lines up with the AOI title
<img width="462" alt="screen shot 2018-01-31 at 4 12 24 pm" src="https://user-images.githubusercontent.com/7633670/35648110-9c3c1520-06a2-11e8-9a57-9047fafd8792.png">